pkg/reconciler/ingress/status_test.go

```diff
@@ -316,6 +317,7 @@ func TestProbeLifecycle(t *testing.T) {
 		v1alpha1.IngressVisibilityExternalIP: sets.NewString("default/gateway"),
 	}
 
+	ia := &(*iaTemplate)
```

Comment:
Same here. `&(*)` is not a full deep copy, it will only work at the first level.